core/lib/server/pub.ts

```diff
@@ -1354,6 +1374,68 @@ export async function getPubsWithRelatedValuesAndChildren<
 									)
 								)
 								.leftJoin("PubsInStages", "pubs.id", "PubsInStages.pubId")
+								.where((eb) =>
+									eb.exists(
+										eb.selectFrom("capabilities" as any).where((ebb) => {
```

Comment:
kysely/typescript don't like us selecting from the later CTE so "we" (thomas) cast it to any and then use some magic to create an ExpressionBuilder with the right type parameter (all the tables/aliases in scope) so that kysely doesn't complain